app/views/devise/registrations/edit.html.haml

```diff
@@ -6,9 +6,17 @@
   = form_for(resource, as: resource_name, url: registration_path(resource_name), html: { method: :put }) do |f|
     = devise_error_messages!
```

Comment:
Yup, thats how all of our validation errors appear now.  I have a story open to replace the error-handling code with a helper method that returns styled HTML for error display - we can fix this issue there.